src/main/java/eu/decentsoftware/holograms/api/utils/items/SkullUtils.java

```diff
@@ -118,17 +120,32 @@ public static String getTexture(@NonNull ItemStack itemStack) {
 	 * Set the Base64 texture of the given skull ItemStack.
 	 *
 	 * @param itemStack The ItemStack.
-	 * @param texture   The new skull texture (Base64).
+	 * @param texture   The new skull texture (Base64 or URL).
 	 */
 	public static void setSkullTexture(@NonNull ItemStack itemStack, @NonNull String texture) {
 		try {
 			ItemMeta meta = itemStack.getItemMeta();
 			if (meta instanceof SkullMeta) {
 				GameProfile profile = new GameProfile(UUID.randomUUID(), "");
-				Property property = new Property("textures", texture);
 
-				PropertyMap properties = profile.getProperties();
-				properties.put("textures", property);
+				String json;
+				
+				try{
+					json = Base64.getDecoder().decode(texture);
+				}
+				catch(IllegalArgumentException ex){
+					json = "invalid";
+				}
+				if(!json.contains("\"url\":\"http://textures.minecraft.net/texture/")){
+					// Decoded json does not contain a valid skin URL, so check if the 'texture' argument is a valid URL.
+					if(texture.startsWith("http://textures.minecraft.net/texture/")); // We are all set.
```

Comment:
I think this check should be done before trying to decode `texture` as json. You actually don't really need to decode it at all. Just check if `texture` is a URL, and if it is, do whatever you do here. Otherwise just skip this and don't do anything.